client/my-sites/stats/hooks/use-stats-navigation-history.ts

```diff
@@ -89,30 +115,41 @@ export const useStatsNavigationHistory = (): { text: string; url: string | null
 		queryParams: {},
 		period: 'day',
 	} );
-	const siteId = useSelector( getSelectedSiteId );
-	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
 
 	useEffect( () => {
 		try {
-			const navState = JSON.parse( sessionStorage.getItem( STORAGE_KEY ) || '[]' );
+			const args = getQueryArgs( window.location.search );
```

Comment:
now I can see why it works.

related: https://github.com/Automattic/jetpack/pull/43567/files#r2110495312